js/Timer.js

```diff
@@ -0,0 +1,137 @@
+      deadline = new Date(today.getFullYear(), today.getMonth(), today.getDate(), 11, 49 , 0, 0);
+      currentblock = 6;
+    }
+    else if(((today.getHours() === 11) && (today.getMinutes() >= 52)) || ((today.getHours() === 12) && (today.getMinutes() < 47))){
+      deadline = new Date(today.getFullYear(), today.getMonth(), today.getDate(), 12, 47 , 0, 0);
+      currentblock = 7;
+    }
+    else if(((today.getHours() === 12 ) && (today.getMinutes() >= 47)) || ((today.getHours() === 13) && (today.getMinutes() < 17))){
+      deadline = new Date(today.getFullYear(), today.getMonth(), today.getDate(), 13, 17 , 0, 0);
+      currentblock = 0;
+    }
+    else if(((today.getHours()===13)&&(today.getMinutes()>20))||(((today.getHours()>14)) && (today.getMinutes() < 11))){
+      deadline = new Date(today.getFullYear(), today.getMonth(), (today.getDate()), 14, 11, 0, 0);
+      currentblock = 8;
+    }
+    else if(((today.getHours()===14)&&(today.getMinutes()>11))||((today.getHours()>=14))){
+      deadline = new Date(today.getFullYear(), today.getMonth(), (today.getDate()+1), 7, 45, 0, 0);
+      currentblock = 9;
+    }
+    return deadline;
+  }
+
+  var currentDay = new Date();
+  var endBlock = new Date();
+  if(currentDay.getDay()===2){
+    endBlock = advisoryDay(currentDay);
+  }else{
+    endBlock = normalDay(currentDay);
+  }
+  return endBlock;
+}
+
+function updateEndBlock(){
+  var t = Date.parse(getEndBlock()) - Date.parse(new Date());
+  var Lseconds = Math.floor((t / 1000) % 60);
+  var Lminutes = Math.floor((t / 1000 / 60) % 60);
+  var Lhours = Math.floor((t / (1000 * 60 * 60)) % 24);
+  //var Ldays = Math.floor(t / (1000 * 60 * 60 * 24));
+  var Blocks=[
+    ["Advisory", "A Block", "B Block", "C Block", "First Lunch (D Block)", "Second Lunch (D Block)", "Third Lunch (D Block)", "E Block", "F Block"],
+    ["Advisory", "A Block", "D Block", "B Block", "First Lunch (F Block)", "Second Lunch (F Block)", "Third Lunch (F Block)", "G Block", "E Block"],
+    ["Advisory", "A Block", "B Block", "C Block", "First Lunch (E Block)", "Second Lunch (E Block)", "Third Lunch (E Block)", "D Block", "G Block"],
+    ["Advisory", "A Block", "C Block", "D Block", "First Lunch (G Block)", "Second Lunch (G Block)", "Third Lunch (G Block)", "E Block", "F Block"],
+    ["Advisory", "A Block", "B Block", "C Block", "First Lunch (F Block)", "Second Lunch (F Block)", "Third Lunch (F Block)", "G Block", "D Block"],
+    ["Advisory", "A Block", "C Block", "B Block", "First Lunch (G Block)", "Second Lunch (G Block)", "Third Lunch (G Block)", "F Block", "E Block"],
+    ["Advisory", "B Block", "C Block", "D Block", "First Lunch (E Block)", "Second Lunch (E Block)", "Third Lunch (E Block)", "F Block", "G Block"],
+  ];
+
+  document.getElementById("blockTime").innerHTML = Lhours + "h " + Lminutes + "m " + Lseconds + "s ";
+
+  if (dy ===0){
+    document.getElementById("blockTime").innerHTML = "Timer not available.";
```

Comment:
Get rid of the period and do a descriptive message with most likely error, like 'No School'. Or a countdown until Monday?